### PR TITLE
Fix: Removed the onBackPressed Bug in TransferProcessFragment

### LINKED ISSUE #665: Wrong behaviour of onBackPressed ( )
**Summary:**
When money is transferred from one account to another , a screen pops up on successful transaction which says that transfer is successful . When we select the close button present at the bottom of the screen we are directed to **SavingsAccountDetails** fragment which is correct.  But when the back button is pressed or the the back arrow on the toolbar is pressed , we are directed to  **SavingsMakeTransfer** Fragment which should not be the case.There should be consistency in the process.

**Steps to reproduce:**
1) Go to Accounts
2) Select an of the mentioned savings account
3) Select **Deposit**
4) Transfer the required money.

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixes #665 
device-2018-03-13-144249

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by miPlodder** [comment]
@sridharjajoo Please review Checkstyle errors from link below and update the PR.

https://travis-ci.org/openMF/mifos-mobile/builds/352745365#L2888

**Comment by sridharjajoo** [comment]
@therajanmaurya @dilpreet96 Kindly review!

File: app/src/main/java/org/mifos/mobilebanking/ui/fragments/TransferProcessFragment.java [hunk 1]
```diff
@@ -147,6 +149,7 @@ public void closeClicked() {
         getActivity().getSupportFragmentManager().popBackStack();
     }
 
+
```
Comment (therajanmaurya) [inline app/src/main/java/org/mifos/mobilebanking/ui/fragments/TransferProcessFragment.java]: Remove this blank line.

File: app/src/main/java/org/mifos/mobilebanking/ui/fragments/TransferProcessFragment.java [hunk 2]
```diff
@@ -79,6 +80,7 @@ public static TransferProcessFragment newInstance(TransferPayload payload, Trans
         return fragment;
     }
 
+
```
Comment (therajanmaurya) [inline app/src/main/java/org/mifos/mobilebanking/ui/fragments/TransferProcessFragment.java]: Remove this blank line.

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by therajanmaurya** [comment]
@sridharjajoo Please follow commit guideline. you can find guideline doc in wiki.

**Comment by sridharjajoo** [comment]
@therajanmaurya Thanks! Will take care of that in future.

**Comment by sridharjajoo** [comment]
@therajanmaurya Done the changes!

**Comment by therajanmaurya** [comment]
@sridharjajoo your commit message supposed to be this 
`Fix: Removed the onBackPressed Bug in TransferProcessFragment`

**Comment by sridharjajoo** [comment]
@therajanmaurya @dilpreet96 Kindly review this P.R.

**Comment by sridharjajoo** [comment]
@therajanmaurya Kindly review!

**Pull request merged**